components/ui-api-layer/internal/gqlschema/schema.graphql

```diff
@@ -63,6 +69,16 @@ type ServiceInstanceEvent {
     instance: ServiceInstance!
```

Comment:
To be consistent it should be `serviceInstance`. Please check also in other `event` types.